### ISSUE TITLE
Omit specific projects

### ISSUE THREAD

**Issue by nintoantony** [tom-englert/ResXResourceManager#128]
Is there any way to omit a specific project when exporting using powershell script.

**Comment by tom-englert** [comment]
Yes, as you can see in the sample you can pass the items to export:

	# Export to excel; parameters are <file name>, [<items>, [<languag(es) of values>, [<languag(es) of comments>, [<export mode>]]]]
	# All parameters except file name are optional:
	#  items:				  default = $null (export all items)
	#  languages of values:   default = $null (export all languages)
	#  languages of comments: default = $null (export no comments)
	#  export mode:           default = 0 (single sheet), 1 = multiple sheets
	"Export changes since last snapshot to Excel"
	$myhost.ExportExcel($excelFileName, $changes, @($neutralCulture, $outputCulture))

**Comment by tom-englert** [comment]
@nintoantony did this answer your question?
